mobile-app/app/screens/AppNavigator/screens/Balances/BalancesScreen.tsx

```diff
@@ -124,19 +114,18 @@ export function BalancesScreen ({ navigation }: Props): JSX.Element {
   }, [getTokenPrice, tokens])
 
   const totalLockedUSDValue = useMemo(() => {
-     return vaults.reduce((totalLockedUSDValue: BigNumber, vault: LoanVault) => {
-      if (vault.state === LoanVaultState.IN_LIQUIDATION) {
-        return totalLockedUSDValue
-      }
-      const totalCollateralUSDAmount: BigNumber = vault.collateralAmounts.reduce(
-        (totalCollateralUSDAmount: BigNumber, token: LoanVaultTokenAmount) => {
-        const usdAmount = getTokenPrice(token.symbol, new BigNumber(token.amount))
-          return totalCollateralUSDAmount.plus(usdAmount.isNaN() ? 0 : usdAmount)
-      }, new BigNumber(0))
-
-      return totalLockedUSDValue.plus(totalCollateralUSDAmount.isNaN() ? 0 : totalCollateralUSDAmount)
+    if (lockedTokens === undefined) {
+      return new BigNumber(0)
+    }
+    return [...lockedTokens.keys()]
```

Comment:
No need to spread right?

Reply:
Would map.values work? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/values

Reply:
`lockedTokens.keys()` gives us object in return, to convert it into array we used spread operator